validation/connection_fields.xsd

```diff
@@ -49,6 +55,7 @@
       <xs:enumeration value="selection"/>
       <xs:enumeration value="boolean"/>
       <xs:enumeration value="file"/>
+      <xs:enumeration value="textbox"/>
```

Comment:
Looks like we were missing the type in packager and only had updated in the monolith.